coss/opensource_clubs/models.py

```diff
@@ -158,6 +158,45 @@ class FAQPage(Page):
     ]
 
 
+class ResourcesPage(Page):
+    """Resources Page for Open Source Clubs."""
+    heading_text = fields.RichTextField(verbose_name='Text', blank=True)
+    heading_image = models.ForeignKey(
+        'wagtailimages.Image',
+        null=True,
+        blank=True,
+        on_delete=models.SET_NULL,
+        related_name='+',
+        verbose_name='Image'
+    )
+    mentors = fields.StreamField([
+        ('info', InputTextContentBlock(required=True),),
+    ])
+    resources_title = models.CharField(verbose_name='Title', blank=True, default='',
+                                       max_length=50)
+    resources_cta_text = models.CharField(verbose_name='CTA Text', blank=True, default='',
+                                          max_length=50)
+    resources_cta_link = models.URLField(verbose_name='Link', blank=True, default='')
+    guides = fields.RichTextField(verbose_name='Guides', blank=True)
+
+    def get_mentors(self):
+        return ClubProfile.objects.filter(is_mentor=True)
```

Comment:
Let's add a todo here to filter the users that belong to each club.

Reply:
Do we need that in this specific method? Resources page content scope is for all clubs and mentors listed here are for all Club members. See also below.

Reply:
I believe we do. I suspect that different clubs will have different mentors. That said, we cannot do much at the moment so let's add a `TODO` as a placeholder.